docs/self-hosting/govern/integrations/github.md

```
@@ -36,130 +36,137 @@ To configure GitHub integration, you'll need to create a GitHub App within your
 
 == GitHub Cloud {#github-cloud}
 
-Follow these steps to create a GitHub App, set callback URLs, and configure webhooks so Plane can sync PRs and commits from GitHub Cloud.
+   Follow these steps to create a GitHub App, set callback URLs, and configure webhooks so Plane can sync PRs and commits from GitHub Cloud.
 
-#### GitHub Cloud
+   #### GitHub Cloud
```

markdownlint-cli2 (0.22.0)

[warning] Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)